korge/src/commonTest/kotlin/com/soywiz/korge/view/SpritesTest.kt

```diff
@@ -0,0 +1,89 @@
+package com.soywiz.korge.view
+
+import com.soywiz.klock.*
+import com.soywiz.korge.tests.*
+import com.soywiz.korge.tiled.*
+import com.soywiz.korge.view.*
+import com.soywiz.korim.bitmap.*
+import com.soywiz.korim.color.*
+import com.soywiz.korim.format.readBitmap
+import com.soywiz.korim.format.writeBitmap
+import com.soywiz.korio.async.*
+import com.soywiz.korio.file.std.*
+import com.soywiz.korio.lang.*
+import com.soywiz.korma.geom.*
+import kotlinx.coroutines.*
+import kotlin.jvm.*
+import kotlin.test.*
+
+class SpritesTest : ViewsForTesting() {
+    lateinit var countDownSpriteMap : Bitmap32
+    lateinit var digits: Array<Bitmap>
+    lateinit var countDownAnimation : SpriteAnimation
+    lateinit var countDownSprite : Sprite
+
+    // MUST BE CALLED BEFORE EACH TEST
+    suspend fun setup(){
+        async(Dispatchers.Default){
```

Comment:
Minor: suspend fun without async + await doesn't directly work?

Reply:
I thought without something to `.await()` the tests could run before the init code is actually completed

Reply:
```
suspend fun() {
async {
    // code
}.await()
}
```

is equivalent to:

```
suspend fun() {
    // code
}
```

Since calling to suspend functions aware already suspending the function, launch/await are there to parallelize (doing the opposite)

Reply:
oh damn you're right of course. I got weirded out getting the `@Before` to work